amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2495,7 +2497,193 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  ##retention-schedule
+  '/categories/{categoryId}/retention-schedules':
```

Comment:
This is not consistent with the existing endpoints. The first part should definitely say `record-categories`.

Reply:
done the changes as per the review comment.